src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -390,6 +391,7 @@ private void addOperations(Set<String> methodSignatures) throws OperationParseEx
    *   Note: if you add to this list, also update the Javadoc for check_object_contracts.
    */
   private void addDefaultContracts() {
+    contracts.add(EqualsTransitive.getInstance());
     contracts.add(new EqualsReflexive());
```

Comment:
Should other contracts should be singletons, too?
If so, could you create a separate pull request that makes that change?